tests/subsys/storage/flash_mpa/src/main.c

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
Typo in directory:
` tests/subsys/storage/flash_mpa` -> ` tests/subsys/storage/flash_map`